package.json

```diff
@@ -46,7 +46,7 @@
     "tailwindcss": "3.4.3",
     "typescript": "5.4.5"
   },
-  "packageManager": "pnpm@9.0.6",
+  "packageManager": "pnpm@9.1.0",
```

Comment:
Er på telefonen nå, så ikke helt sikker. Men er det ikke noe greier vi kan sette i pnpm configen som sier at denne ikke er strict? 

Eventuelt kan verdien i package.json endres til å ikke være strict? Om jeg er på versjon 9.1.0 eller 9.1.1 er ikke så interessant tenker jeg. Det viktigste er at man ikke bruker pnpm 8!

Reply:
Ja, jeg var litt inne på samme tanke. Er litt slitsomt å måtte oppdatere den hele for hver enste minor og patch. Man kan vel ofte skrive ranges i `package.json`, men ser ikke ut som det støttes enda: [https://github.com/nodejs/corepack/issues/208](https://github.com/nodejs/corepack/issues/208) (markert som duplikat av [https://github.com/nodejs/corepack/issues/95](https://github.com/nodejs/corepack/issues/95), som fortsatt er åpen)